vtexdocs/devportal
# Feature/Markdown Rendering

#### What is the purpose of this pull request?

To add markdown rendering support to the Developers Portal.
Documented [here](https://www.notion.so/vtexhandbook/Adicionar-rota-ao-card-API-Guides-da-landing-page-com-renderiza-o-padr-o-de-arquivos-markdown-0dddc3ae5e03429b8eebb7bbf0d45e16)

#### What problem is this solving?

Currently, there are no pages in the Developers Portal rendering markdown files. When the user clicks the API Guides card, both on the Documentation section of the landing page and on the Docs dropdown menu of the header, they are redirected to a page containing only the sidebar and a text that says "Hello, world".

#### How should this be manually tested?

Go to [this page](https://deploy-preview-29--xenodochial-shockley-a99d82.netlify.app/) and try accessing the route `/docs/api-guides` both through the API Guides card in the documentation section and the API Guides button inside the Docs dropdown menu of the header. You should see a page with the sidebar and links to other pages in both situations. Also, try clicking those links. You should be redirected to a page that renders the correspondent markdown file (`/public/docs/api-guides/billing-options.md` for the Billing Options link and `/public/docs/api-guides/clients.md` for the Clients link).

#### Screenshots or example usage

#### Types of changes
